examples/gallery/plot/image.py

```diff
@@ -0,0 +1,29 @@
+"""
+Images or EPS files on maps
+---------------------------
+The :meth:`pygmt.Figure.image` method can be used to read and 
+place a raster image file or an Encapsulated PostScript file
+on a map. We must specify the file as *str* via the ``imagefile`` 
+argument or simply use the filename as first argument. You can 
+also use a full URL pointing to your desired image. The ``position`` 
+argument allows us to set a reference point on the map for the image.
+
+For more advanced style options, see the full option list 
+at :gmt-docs:`image.html`.
+"""
+
+import pygmt
+
+fig = pygmt.Figure()
+
+fig.basemap(region=[0, 2, 0, 2], projection="x3c", frame=True)
```

Comment:
I feel that `X6c` is more readable than `x3c`.
```suggestion
fig.basemap(region=[0, 2, 0, 2], projection="X6c", frame=True)
```